examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -2024,6 +2024,21 @@
               "maxInterval": 65534,
               "reportableChange": 0
             },
+            {
+              "name": "SupportsConcurrentConnection",
+              "code": 4,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "external",
```

Comment:
```suggestion
              "storageOption": "External",
```
and similar for other ZAP files.

Reply:
Why did ZAP not complain?

Reply:
Let's see... When the info from the JSON is loaded into the database, the object representing the attribute is passed to `importAttributeForEndpointType` which just dumps whatever string is there in the DB.

Then later the endpoint_config generator does:
```
        if (a.storage == dbEnum.storageOption.nvm) {
          mask.push('TOKENIZE')
        }
        if (a.storage == dbEnum.storageOption.external) {
          mask.push('EXTERNAL_STORAGE')
        }
```
which is probably where we should be doing a check and failing out if the value is not ....